krun/util.py

```diff
@@ -180,3 +181,15 @@ def print_session_info(config):
             print("  %s" % k)
     else:
         print("  No keys skipped")
+
+
+def make_heat():
+    """A loop which spins, attempting to make heat.
+
+    Used when the CPU is too cool to run a benchmark."""
+
+    until = time.clock() + 1  # in roughly one second
+    while True:
+        # Waste cycles
+        if time.clock() > until:
```

Comment:
Yep, I agree that any no-op will execute for a variable time. Fortunately, in this case, we really don't care about that too much.

Does krun run under cpython or pypy? If the latter, I think that `for` loop will be entirely optimised away. [You'd need to do something like have a variable `j = 0` outside it, to which you add 1 on each iteration of the loop.]